enterprise/migrations/0130_lms_customer_lp_search_help_text.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 2.2.20 on 2021-05-03 18:30
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('enterprise', '0129_enterprisecatalogquery_uuid_unique'),
+    ]
+
+    operations = [
+        migrations.AlterField(
```

Comment:
The `AlterField`'s here are not changing the actual field name to do this rename for the Django Admin, which means we can avoid doing a 3-step release (per our migrations docs for renaming model fields). Rather, it only adds a verbose name and updates the help text.